main.go

```diff
@@ -79,72 +80,72 @@ func ndkHome() string {
 	return "ndk-bundle"
 }
 
-func inPath(path string) bool {
-	return strings.Contains(os.Getenv("PATH"), path)
+func newNDKHome() (string, error) {
+	if v := os.Getenv("ANDROID_HOME"); v != "" {
+		// $ANDROID_HOME is deprecated
+		return filepath.Join(v, "ndk-bundle"), nil
+	}
+	if v := os.Getenv("ANDROID_SDK_ROOT"); v != "" {
+		// $ANDROID_SDK_ROOT is preferred over $ANDROID_HOME
+		return filepath.Join(v, "ndk-bundle"), nil
+	}
+	return "", errors.New("neither $ANDROID_HOME nor $ANDROID_SDK_ROOT is specified")
 }
 
 func updateNDK(revision string) error {
```

Comment:
I think one key information is missing here that cannot be inferred from the code alone: that we intend to install the new NDK (if needed at all) to `ndk-bundle` and set `ANDROID_NDK_HOME` to that folder to keep compatibility with all AGP versions.

Should we add a comment to clarify this?